main/src/ca/uwaterloo/flix/language/ast/Rigidity.scala

```diff
@@ -17,4 +19,14 @@ object Rigidity {
     */
   case object Rigid extends Rigidity
 
+  /**
+    * The type of a rigidity environment.
+    */
+  type Env = SortedSet[Symbol.KindedTypeVarSym]
```

Comment:
We usually don't use type aliases. But okay.

Reply:
I'll replace it with a proper type